astropy/time/tests/test_comparisons.py

```diff
@@ -30,8 +30,8 @@ def test_miscompares(self):
                 op(t1, None)
             assert str(err).endswith("Unsupported operand type(s) for {0}: 'Time' and 'NoneType'"
                                      .format(op_str))
-        assert (t1 == None) is False
-        assert (t1 != None) is True
+        assert (t1 is None) is False
+        assert (t1 is not None) is True
```

Comment:
This just looks like a duplicate to me. Maybe the original author can explain why both comparisons are needed.